lib/cc/config.rb

```diff
@@ -19,7 +19,9 @@ class Config
 
     def self.load
       config = Default.new
-      config = config.merge(YAML.new) if File.exist?(YAML::DEFAULT_PATH)
+      if File.exist?(YAML::DEFAULT_PATH) && !File.exist?(JSON::DEFAULT_PATH)
+        config = config.merge(YAML.new)
+      end
       config = config.merge(JSON.new) if File.exist?(JSON::DEFAULT_PATH)
```

Comment:
I think this would be more readable like:

```rb
if File.exists?(JSON::DEFAULT_PATH)
  config.merge(JSON.new)
elsif File.exists?(YAML::DEFAULT_PATH)
  config.merge(YAML.new)
end
```